ItalianTranslation_PlanetCoaster.xml

```diff
@@ -14793,14 +14793,14 @@
 		<entry key="Research_Revealed_Scenery_SpecialEffects" translation="Effetti Speciali"/>
 		<entry key="Research_Revealed_Scenery_Western" translation="Scenario Western"/>
 		<entry key="Research_Revealed_ShopsAndFacilities_ATM" translation="Bancomat"/>
-		<entry key="Research_Revealed_ShopsAndFacilities_Balloon" translation="Looney Balloons"/>
+		<entry key="Research_Revealed_ShopsAndFacilities_Balloon" translation="Looney Blooons"/>
 		<entry key="Research_Revealed_ShopsAndFacilities_Burger" translation="Capitan Manzo"/>
 		<entry key="Research_Revealed_ShopsAndFacilities_Chicken" translation="Tikichiki"/>
 		<entry key="Research_Revealed_ShopsAndFacilities_Coffee" translation="Street Fox Coffee"/>
 		<entry key="Research_Revealed_ShopsAndFacilities_FirstAid" translation="Pronto Soccorso"/>
 		<entry key="Research_Revealed_ShopsAndFacilities_Fries" translation="Monsieur Frites"/>
 		<entry key="Research_Revealed_ShopsAndFacilities_Gulpee" translation="Gulpee"/>
-		<entry key="Research_Revealed_ShopsAndFacilities_Hat" translation="Cappelli Fantastici"/>
+		<entry key="Research_Revealed_ShopsAndFacilities_Hat" translation="Hat's Fantastic"/>
```

Comment:
Altre volte questo torna in inglese, forse prima della pull request ti conviene aggiornare il tuo file, se vuoi ti spiego come fare (utilizzando programmi come Notepad++)

Reply:
Come sopra.. é voluto